tests/python/pytorch/graphbolt/test_csc_sampling_graph.py

```diff
@@ -212,6 +212,19 @@ def test_node_type_offset_wrong_legnth(node_type_offset):
         )
 
 
+def check_tensors_on_the_same_shared_memory(t: torch.Tensor, t2: torch.Tensor):
+    """Check if two tensors are on the same shared memory.
+
+    Cannot use `.data_ptr()` here because the memory created by different
```

Comment:
Why shared memory does not share the pointer?

Reply:
I guess it is because the pointers are created by two different `shm_open` calls.